docs/user-guide/introspection-and-debugging.md

```diff
@@ -0,0 +1,296 @@
+
+```bash
+$ kubectl get pods
+NAME             READY     REASON    RESTARTS   AGE
+my-nginx-9unp9   0/1       Pending   0          8s
+my-nginx-b7zs9   0/1       Running   0          8s
+my-nginx-i595c   0/1       Running   0          8s
+my-nginx-iichp   0/1       Running   0          8s
+my-nginx-tc2j9   0/1       Running   0          8s
+```
+
+To find out why the my-nginx-9unp9 pod is not running, we can use ```kubectl describe pod``` on the pending Pod and look at its events:
+
+```bash
+$ kubectl describe pod my-nginx-9unp9 
+Name:				my-nginx-9unp9
+Image(s):			nginx
+Node:				/
+Labels:				app=nginx
+Status:				Pending
+Reason:				
+Message:			
+IP:				
+Replication Controllers:	my-nginx (5/5 replicas created)
+Containers:
+  nginx:
+    Image:	nginx
+    Limits:
+      cpu:		600m
+      memory:		128Mi
+    State:		Waiting
+    Ready:		False
+    Restart Count:	0
+Events:
+  FirstSeen				LastSeen			Count	From		SubobjectPath	Reason			Message
+  Thu, 09 Jul 2015 23:56:21 -0700	Fri, 10 Jul 2015 00:01:30 -0700	21	{scheduler }			failedScheduling	Failed for reason PodFitsResources and possibly others
+```
+
+Here you can see the event generated by the scheduler saying that the Pod failed to schedule for reason ```PodFitsResources``` (and possibly others). ```PodFitsResources``` means there were not enough resources for the Pod on any of the nodes. Due to the way the event is generated, there may be other reasons as well, hence "and possibly others." 
+
+To correct this situation, you can use ```kubectl stop rc``` to stop your Replication Controller and start a new one with fewer replicas, or you can use ```kubectl update``` to update your Replication Controller to 4 replicas, and then use ```kubectl stop pod``` to kill the pending pod. (If you don’t update the Replication Controller, it will re-create a pod after you kill the pending one.)
+
+In addition to ```kubectl describe pod```, another way to get extra information about a pod (beyond what is provided by ```kubectl get pod```) is to pass the ```-o yaml``` output format flag. This will give you even more information than ```kubectl describe pod```--essentially all of the information the system has about the Pod. Here you will see things like annotations (a kind of label for internal use only), restart policy, ports, and volumes.
```

Comment:
s/(a kind of label for internal use only)/(key-value metadata without the label restrictions)/

?